index.js

```diff
@@ -16,6 +17,7 @@ module.exports = {
     var dep = checker.for('ember-cli', 'npm');
 
     this._shouldSetupRegistryInIncluded = !dep.satisfies('>=0.2.0');
+    this._shouldShowBabelDeprecations = !dep.lt('2.11.0-beta.1');
```

Comment:
I think this should be `< 2.11.0-beta.2` since 2.11.0-beta.1 didn't include the code to prevent deprecations...?